main_test.go

```diff
@@ -11,20 +11,25 @@ import (
 	"github.com/cucumber/godog/colors"
 )
 
+// Mongo version here is overridden in the pipeline by the URL provided in the component.sh
```

Comment:
`overridden … in the component.sh` Is it?
I can't see `component.sh` changes in this PR. This is the equivalent in dataset api…
https://github.com/ONSdigital/dp-dataset-api/blob/develop/ci/scripts/component.sh#L8
Maybe that needs to be added to this repo as well.

Reply:
Good spot - I think this needs adding to the repo generation template

Reply:
Difficult, as it is only used if the component has mongo. 🤔

Reply:
On second thoughts maybe not, as not all repos will have the mongo connection 🤦